src/core/background/scrobbler/listenbrainz-scrobbler.js

```diff
@@ -269,7 +271,11 @@ define((require) => {
 			const trackMeta = {
 				artist_name: song.getArtist(),
 				track_name: song.getTrack(),
-				additional_info: {},
+				additional_info: {
+					submission_client: 'Web Scrobbler',
+					submission_client_version: UtilBrowser.getExtensionVersion(),
+					music_service: song.metadata.label,
```

Comment:
We reserve the `music_service` field for a canonical representation of the service in question, while `music_service_name` can be used for other cases. Here capitalization etc. may vary without it being an issue
```suggestion
					music_service_name: song.metadata.label,
```